clouddriver-web/src/main/groovy/com/netflix/spinnaker/clouddriver/Main.groovy

```diff
@@ -48,16 +48,22 @@ import java.security.Security
 @EnableScheduling
 class Main extends SpringBootServletInitializer {
 
-  static final Map<String, String> DEFAULT_PROPS = [
+  private static final Map<String, String> DEFAULT_PROPS = [
```

Comment:
this wasn't private because it is referenced in ConfigRefreshController (probably not breaking because thanks groovy)

you should be able to leave this private, delete launchArgs in this class, and delete ConfigRefreshController since spring-cloud-context should already be providing that functionality

Reply:
I was hoping the `ConfigRefreshController` and supporting code could be deleted after this new refresh mechanism was in place, but wasn't sure if I should do that in this PR. I'm happy to delete some more code.